.github/workflows/ci.yml

```diff
@@ -0,0 +1,125 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+      - '[0-9]+.x'
+      - '[0-9]+.[0-9]+.x'
+  pull_request:
+    types: [opened, reopened, synchronize]
```

Comment:
This is usually what you want... [so its the default :)](https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#pull_request)

![image](https://user-images.githubusercontent.com/597907/98592277-7f8ca400-229f-11eb-948d-9d481aa1adf1.png)

It's _not_ a big deal, but this can be simplified slightly and in the future you won't need to lookup the docs if you remember that the default is good:

```suggestion
```